<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Empleados</class>
 <widget class="QDialog" name="Empleados">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="empleado">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="buscar">
      <attribute name="title">
       <string>Buscar</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Buscar por:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBoxEmp">
           <item>
            <property name="text">
             <string>id_empleado</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>nombre</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>telefono</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tienda</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit2"/>
         </item>
         <item>
          <widget class="QPushButton" name="buscar_2">
           <property name="text">
            <string>Buscar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="agregar">
      <attribute name="title">
       <string>Agregar</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="id_empleado">
             <property name="text">
              <string>ID:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="3">
            <widget class="QLineEdit" name="edit_id"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="nombre">
             <property name="text">
              <string>Nombre:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2" colspan="2">
            <widget class="QLineEdit" name="edit_nombre"/>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLabel" name="domicilio">
             <property name="text">
              <string>Domicilio:  </string>
             </property>
            </widget>
           </item>
           <item row="2" column="2" colspan="2">
            <widget class="QLineEdit" name="edit_domicilio"/>
           </item>
           <item row="3" column="0" colspan="3">
            <widget class="QLabel" name="telefono">
             <property name="text">
              <string>Telefono: </string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QLineEdit" name="edit_telefono"/>
           </item>
           <item row="4" column="0" colspan="3">
            <widget class="QLabel" name="puesto">
             <property name="text">
              <string>Puesto:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="3">
            <widget class="QLineEdit" name="edit_puesto"/>
           </item>
           <item row="5" column="0" colspan="3">
            <widget class="QLabel" name="salario">
             <property name="text">
              <string>Salario:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="3">
            <widget class="QLineEdit" name="edit_salario"/>
           </item>
           <item row="6" column="0" colspan="3">
            <widget class="QLabel" name="id_tienda">
             <property name="text">
              <string>Tienda:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="3">
            <widget class="QLineEdit" name="edit_tienda"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="agregar_2">
           <property name="text">
            <string>Agregar</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
